Multisig/UI/Transaction/ExecuteTransaction/ReviewExecutionViewController.swift

```diff
@@ -16,6 +16,9 @@ import SafariServices
 
 class ReviewExecutionViewController: ContainerViewController, PasscodeProtecting {
 
+    static let MIN_RELAY_TXS_LEFT = 0
```

Comment:
Do we need this constant? is it possible to be else than 0?

Reply:
For debugging I like to set it to 5 to force the exceution via EOA. It also helps to find all places where this is checked